keyring/backends/SecretService.py

```diff
@@ -80,7 +80,7 @@ def set_password(self, service, username, password):
             "service": service,
             "username": username
         }
-        label = "Password for '%s' on '%s'" % (username, service)
+        label = "Password for '{}' on '{}'".format(username, service)
```

Comment:
I probably would have deferred these compatible changes for another time, as it's more about style than compatibility, but happy to accept with them.